nuxt.config.js

```diff
@@ -47,7 +48,7 @@ module.exports = {
   /*
   ** Plugins to load before mounting the App
   */
-  plugins: ['~/plugins/vue-filters', '~/plugins/i18n'],
+  plugins: ['~/plugins/vue-filters', '~/plugins/i18n.js'],
```

Comment:
Is this change needed?
Shouldn't it be kept consistent between the two plugins loaded here?

Reply:
It is needed because of the existence of a plugins/i18n directory. Without it there is a warning about ambiguity of what is being required.